doc/README.md

```diff
@@ -237,10 +237,45 @@ them. This is useful to combine profiles from multiple processes of a
 distributed job. The profiles may be from different programs but must be
 compatible (for example, CPU profiles cannot be combined with heap profiles).
 
-pprof can subtract a profile from another in order to compare them. For that,
-use the **-base= _profile_** option, where *profile* is the filename or URL for the
-profile to be subtracted. This may result on some report entries having negative
-values.
+
+## Comparing profiles
+
+pprof can subtract one profile from another, provided the profiles are of
+compatible types. pprof has two options which can be used to specify the
+filename or URL for a profile to be subtracted from the source profile:
+
+* **-diff_base= _profile_:** useful for comparing any two profiles of compatible
+types (i.e. any two heap profiles). Percentages in the output are relative to
+the total of samples in the diff base profile.
+
+* **-base= _profile_:** useful for subtracting a cumulative profile, like a
+[golang block profile](https://golang.org/doc/diagnostics.html#profiling),
+from another cumulative profile collected from the same program at a later time.
+When comparing cumulative profiles collected on the same profile, percentages in
+the output are relative to the the difference between the total for the source
+profile and the total for the base profile.
+
+The following flag can be used when a base profile is specified with either the
```

Comment:
Since there is only one flag listed here, the pattern "The following flag can be used ... : <flag description>" looks a bit heavyweight - this pattern would make sense if there would be multiple flags, but for one it doesn't.

Reply:
Done.